packages/lumx-react/src/components/date-picker/DatePickerField.tsx

```diff
@@ -28,7 +28,7 @@ interface DatePickerFieldProps extends GenericProps {
     minDate?: Date;
 
     /** Value. */
-    value: moment.Moment | undefined;
+    value: Date | moment.Moment | undefined;
```

Comment:
Should also handle compatible string. Since this type begins to be a bit complicated and is used inside different files, we should create a dedicated type and import it in the different files.